docs/computer-vision/computer-vision.mdx

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
All of the CV parts in this PR are from the result of syncing this with main (which had a few pieces added and the changelog)